routes/elevation.py

```diff
@@ -0,0 +1,190 @@
+import asyncio
+from flask import (
+    abort,
```

Comment:
There are a handful of imports here that are not used, accordingly to Pylint:
```
elevation.py:2:0: W0611: Unused abort imported from flask (unused-import)
elevation.py:2:0: W0611: Unused Response imported from flask (unused-import)
elevation.py:2:0: W0611: Unused request imported from flask (unused-import)
elevation.py:2:0: W0611: Unused current_app imported from flask as app (unused-import)
elevation.py:15:0: W0611: Unused fetch_data imported from fetch_data (unused-import)
elevation.py:21:0: W0611: Unused project_latlon imported from validate_request (unused-import)
elevation.py:27:0: W0611: Unused nullify_nodata imported from validate_data (unused-import)
elevation.py:27:0: W0611: Unused postprocess imported from validate_data (unused-import)
```

Let's remove imports that are not used. We can add them back later if/when needed.